lib/ansible/module_utils/basic.py

```diff
@@ -2695,7 +2695,7 @@ def _restore_signal_handlers(self):
 
     def run_command(self, args, check_rc=False, close_fds=True, executable=None, data=None, binary_data=False, path_prefix=None, cwd=None,
                     use_unsafe_shell=False, prompt_regex=None, environ_update=None, umask=None, encoding='utf-8', errors='surrogate_or_strict',
-                    expand_user_and_vars=True, pass_fds=None, before_communicate_callback=None):
+                    expand_user_and_vars=True, pass_fds=None, before_communicate_callback=None, raise_timeouts=False):
```

Comment:
I would love to see API documentation for this parameter.

Reply:
done